driver-core/src/main/com/mongodb/operation/OperationHelper.java

```diff
@@ -77,6 +79,30 @@ public Void apply(final T t) {
         }
     }
 
+    static void checkValidReadConcern(final Connection connection, final ReadConcern readConcern) {
+        if (!serverIsAtLeastVersionThreeDotTwo(connection.getDescription()) && !readConcern.equals(ReadConcern.DEFAULT)) {
```

Comment:
I misread - all operations do the null checking, so this is inline with all other operationHelper methods.